parser.rb

```diff
@@ -1,13 +1,28 @@
 require 'pry'
 
 def kinda_like?(fuzzy_word, exact_word)
-  # implement with your code here
-  # breaking the method up into pieces is encouraged
+  fuzzy = fuzzy_word.chars
+  exact = exact_word.chars
+  difference_counter = (fuzzy.length - exact.length).abs
+  common_length = fuzzy.length > exact.length ? exact.length : fuzzy.length
+  (0...common_length).each do |i|
+    difference_counter += 1 if fuzzy[i] != exact[i]
+  end
+  if difference_counter.between?(2, 3) && difference_counter < common_length
```

Comment:
Whenever these get a little complicated (one `&&` is enough), try to extract this combination of factors to a private method, so you can give it a name.